src/components/CreateDataRequestForm.vue

```diff
@@ -94,9 +86,12 @@ export default {
   },
   data() {
     const maxNumber = (rule, value, callback) => {
+      const maxNumber = 18446744073709551615
```

Comment:
This code is repeated twice, it should be extracted into a function like `isSmallerThan2ToThePowerOf64`, but with a better name